### PR TITLE
1726 FETP Graduate Area of Expertise map overlays

### PR DESCRIPTION
**Pull request by avaek**
Addresses: https://github.com/beyondessential/tupaia-backlog/issues/1726

The requirements of this map overlay have been changed since this card was originally made, you can read through the comments for context if you like but long story short the type of map overlay this now creates is screenshotted in this comment (district level map overlay which sums answers to various questions submitted by `individual`'s in that district): https://github.com/beyondessential/tupaia-backlog/issues/1726#issuecomment-778932323

**Note:**
- This branch is branched from https://github.com/beyondessential/tupaia/pull/1939 because my map overlays use `binary` internal data (which has been changed by that branch so I branched off it so these new map overlays will work once that code is released)

### CODE REVIEW

File: packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+// this contains shared properties across all overlays
+const BASE_OVERLAY = {
+  userGroup: 'FETP Graduates',
+  isDataRegional: true,
+  measureBuilderConfig: {
+    aggregationType: 'SUM',
+    entityAggregation: {
+      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
+      dataSourceEntityType: 'individual',
+      aggregationEntityType: 'district',
```
Comment (chris-bes) [inline packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js:31]: I think you can also change this to below and it still gives the same result! (but no need to change it)
```suggestion
    entityAggregation: {
      aggregationType: 'SUM_PER_ORG_GROUP',
      dataSourceEntityType: 'individual',
      aggregationEntityType: 'district',
```

File: packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    name: 'Village and family health',
+    dataElementCode: 'FETPNG20Data_045',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Social_mobilization',
+    name: 'Social mobilization',
+    dataElementCode: 'FETPNG20Data_046',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_	Health_promotion',
```
Comment (chris-bes) [inline packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js:152]: I assume this is a mistake 😸 
```suggestion
    id: 'FETPNG_Grad_Expertise_Health_promotion',
```

**Review by chris-bes — APPROVED** [review]
LGTM! Just need to remove the extra spaces so pre approve!

**Comment by avaek** [comment]
Thanks for the fast review @chris-bes & nice catch on that extra space, it was indeed a mistake

**Comment by avaek** [comment]
Closing this PR / deleting this branch cos of merge conflicts, it's easier to just make a fresh PR branched off the latest dev

**Pull request closed without merging**